obs-visca-control.lua

```diff
@@ -220,6 +255,15 @@ function script_load(settings)
         { name = "tilt_down", descr = "Tilt Down", action = actions.Tilt_Down},
         { name = "zoom_in", descr = "Zoom In", action = actions.Zoom_In},
         { name = "zoom_out", descr = "Zoom Out", action = actions.Zoom_Out },
+        { name = "preset_1", descr = "Preset 1", action = actions.Preset_1 },
```

Comment:
In order to use `actions.Preset_Recal`, add a `preset` entry to this table.
```suggestion
        { name = "preset_1", descr = "Preset 1", action = actions.Preset_Recal, action_args = { preset = 1 } },
```
The action_args then need to be passed to `cb_camera_hotkey`.